src/content/blog/astro-4120.mdx

```diff
@@ -0,0 +1,140 @@
+
+With **Server Islands** we are extending this same architecture to the server side as well. Within any given web page you might have content that is:
+
+- Completely static and never changes.
+- Dynamically backed by a database that changes infrequently, but more often than you deploy.
+- Personalized content, tailored to individual users.
+
+Today you must choose 1 caching strategy for all of these types of content, and if the page is a logged-in experience that usually means no caching at all. Server Islands allow you to mark certain components as being islands. Doing so will **defer** them from loading until the browser has first downloaded the shell. 
+
+<BlogContentImage src={islandDiag} alt="A diagram showing the server island population parts of the page from the server." />
+
+The server islands are used for your most dynamic content; personalized content like the user's avatar, their shopping cart, and product reviews. With these components deferred you're free to more aggressively cache the page itself.
+
+This means that users, logged in or not, will see the most critical parts of the page instantly, as they are caching on Edge CDNs. With careful design considerations the fallback content will be visible for a short amount of time before the dynamic islands are then loaded.
+
+Each island is loaded independent from the rest; this means a slower island, such as one connected to a legacy backend, won't delay the rest of personalized content from being seen and interacted with.
+
+### Trying out Server Islands
+
+We're still working on polishing the feature, but you can try out Astro Server Islands today by enabling the feature in Astro 4.12:
+
+```js
+import { defineConfig } from 'astro/config';
+import netlify from '@astrojs/netlify';
+
+export default defineConfig({
+  output: 'hybrid',
+  adapter: netlify(),
+
+  experimental: {
+    serverIslands: true,
+  },
+})
+```
+
+Server islands, as the name implies, run on the server, so you'll need to use either `'server'` or `'hybrid'` output. Once you've enabled the feature you can use them on any component using the `server:defer` directive:
+
+```astro
+<Avatar server:defer>
+  <GenericUserImage slot="fallback" />
+</Avatar>
+```
+
+When Astro builds your site it will omit the component and inject a script in its place, in addition to the content you've marked with `slot="fallback"`. When the page loads in the browser these components will be requested, along with the props you passed to them, to a special endpoint that renders them and returns the HTML.
+
+Server Islands are normal Astro components and can use any of the features you expect from Astro like middleware, slots, `client:` islands.
```

Comment:
With this sentence I understood: you can use `server:defer` and `client:load` at the same time

```astro
<Foo server:defer client:load />
```
Is that what you wanted to say?

Reply:
No, just that you can use it in general. I think there's currently a bug with using them at the same time.